packages/playwright/src/common/test.ts

```diff
@@ -363,4 +368,8 @@ export class TestCase extends Base implements reporterTypes.TestCase {
     path.push(...this._tags);
     return path.join(' ');
   }
+
+  staticAnnotations(): Annotation[] {
+    return this._staticAnnotations.concat(this.parent.staticAnnotations());
```

Comment:
There is already similar code in `suiteUtils`. Perhaps we get rid of either this or that one? Also, it looks like there is no need to parse/serialize `annotations` in this file, because these are the same as `staticAnnotations`? Perhaps we can just merge the two?

Reply:
How do you like https://github.com/microsoft/playwright/pull/35292/commits/2169d95bb835c8255bda331c8c324362b21c31db?

Reply:
That's better! However, we still have `Test._staticAnnotations` and `Test.annotations` which are the same, aren't they?